run_model.py

```diff
@@ -0,0 +1,23 @@
+from model import Model
+
+
+train_path = "data/curated_1/train_curated_1_final/"
```

Comment:
Would it be better to remove the "_1_final" parts from it? Is it necessary to keep them, or just useless characters?